Lib/test/test_zipfile.py

```diff
@@ -1718,6 +1719,14 @@ def test_non_existent_file_raises_OSError(self):
         # quickly.
         self.assertRaises(OSError, zipfile.ZipFile, TESTFN)
 
+    def test_issue44439(self):
+        q = array.array('Q', [1, 2, 3, 4, 5])
+        LENGTH = len(q) * q.itemsize
+        with zipfile.ZipFile(io.BytesIO(), 'w') as zip:
+            with zip.open('data', 'w') as data:
+                self.assertEqual(data.write(q), LENGTH)
+                self.assertEqual(data._file_size, LENGTH)
```

Comment:
`_file_size` is a private attribute. It is better to use a public API in tests. For example read the content of the file.